package-lock.json

```diff
@@ -62,7 +62,7 @@
     },
     "__tests__/__packages__/cli-test-utils": {
       "name": "@zowe/cli-test-utils",
-      "version": "7.0.0-next.202108121907",
+      "version": "7.0.0-next.202108202027",
```

Comment:
Should we update the changelog for the `@zowe/cli-test-utils` package?

Reply:
Since the @next release should have had to compile successfully during its last PR and promotion into @next, it is not clear whether this change will be apparent to any consumer.

I can add a Change log entry to the effect of 'Fixed compile error'. What do you think?

Reply:
It may be apparent to consumers of the cli-test-utils package (CLI plug-in developers), if they experience the same breakage that we did 🙂

The current changelog update looks good to me, I think there may be a bug in the GitHub workflow